core/index.js

```diff
@@ -6,13 +6,28 @@ console.log('hello World');
 
 //6. Blocking code
  // Run it first and observe the output
-//  var file = fs.readfileSync('./theory.md');
+//   var file = fs.readfileSync('./theory.md');
 //  console.log(file);
-//  console.log('run me first');
+//   console.log('run me first');
  
  // change it to run it in non-blocking manner.
  // Observe the difference in blocking and non-blocking code. 
+ var fs = require("fs");
+ var read = fs.readFile("./theory.md",(err,content)=>{
+   if(err){
+     console.log(err);
```

Comment:
Always add return statement in the error block, otherwise it will continue execution after error occurs.
```js
if(err) return console.log(err);